bugbug/models/worksforme.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class WorksForMeModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        feature_extractors = [
+            bug_features.HasSTR(),
+            bug_features.HasRegressionRange(),
+            bug_features.Status(),
+            bug_features.Severity(),
+            bug_features.Priority(),
+            bug_features.HasCrashSignature(),
+            bug_features.HasURL(),
+            bug_features.Whiteboard(),
+            bug_features.Product(),
+            bug_features.Component(),
+            bug_features.Keywords(),
+            bug_features.TimeToClose(),
+            bug_features.HasPendingNeedinfoOnReporter(),
+        ]
+
+        cleanup_functions = [
+            feature_cleanup.fileref(),
+            feature_cleanup.url(),
+            feature_cleanup.synonyms(),
+        ]
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "bug_extractor",
+                    bug_features.BugExtractor(feature_extractors, cleanup_functions),
```

Comment:
We may want to rollback until before closing the bug.

Reply:
> We may want to rollback until before closing the bug.

@suhaibmujahid Rolling back to before closing the bug makes it impossible to use the HasPendingNeedinfoOnReporter feature.

```
    flag["name"] == "needinfo" and flag["requestee"] == reporter_email
KeyError: 'requestee'
```

Reply:
It seems that the rollback does not support needinfo for now: 

https://github.com/mozilla/bugbug/blob/9b2638c6b6b3af3966c25dd82ff3e8a213e4fcc9/bugbug/bug_snapshot.py#L665

Reply:
Let us try without the `HasPendingNeedinfoOnReporter` feature.